user/models.py

```diff
@@ -44,5 +63,12 @@ def __str__(self):
         return str(self.username or 'Unknown')
 
     def save(self, *args, **kwargs):
+        try:
+            User.objects.get(username=self.username)
```

Comment:
The username is marked as unique and with the username we create the slug.
If I uniquify the slug, I have to uniquify the username again because it will not work. That's why I only uniquify the username.